frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx

```diff
@@ -0,0 +1,806 @@
+                )}
+                className="program__image program__image--woman-smiling"
+                alt=""
+                data-h2-margin="b(top, xxs)"
+                data-h2-position="b(relative)"
+                data-h2-shadow="b(xs)"
+              />
+              <div
+                className="circle circle--top-right"
+                data-h2-position="b(absolute)"
+                data-h2-bg-color="b(ia-pink)"
+              />
+              <div
+                className="circle circle--lg circle--bottom-left"
+                data-h2-position="b(absolute)"
+                data-h2-bg-color="b(ia-purple)"
+              />
+              <img
+                src={imageUrl(INDIGENOUSAPPRENTICESHIP_APP_DIR, "feathers.png")}
+                className="about-program__feathers"
+                alt=""
+                data-h2-position="b(absolute)"
+              />
+            </div>
+            <div className="program__content">
+              <Heading
+                className="about-program__title"
+                data-h2-font-size="b(h3)"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "About the Program",
+                  description: "Program information section title",
+                })}
+              </Heading>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The IT Apprenticeship Program for Indigenous Peoples is a Government of Canada initiative specifically for First Nations, Inuit, and Métis peoples. It is pathway to employment in the federal public service for Indigenous peoples who have a passion for Information Technology (IT).",
+                  description: "First paragraph about the program",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "By valuing and focusing on a person’s potential, rather than on their educational attainment level, the Program removes one of the biggest barriers that exists when it comes to employment within the digital economy. The Program has been developed by, with, and for Indigenous peoples from across Canada.  Its design incorporates the preferences and needs of Indigenous learners while recognizing the importance of community.",
+                  description: "Second paragraph about the program",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Apprentices who are involved in the program say that it is “life-changing”, that it represents “a chance to have a better life through technology”, and that “there are no barriers to succeeding in this program”.",
+                  description: "Third paragraph about the program",
+                })}
+              </p>
+              <div data-h2-visibility="b(invisible) m(visible)">
```

Comment:
These buttons move about between desktop and mobile view but I'm not sure they should.  On desktop view they match the wireframe.  On the mobile view they end up below the "What will I learn" paragraph which is different from the wireframe.  Is this intentional?

Reply:
Yeah, sorry there were some changes made in response to designer feedback and were not reincorporated in the wireframe.